datafusion/src/optimizer/simplify_expressions.rs

```diff
@@ -1191,212 +1178,171 @@ mod tests {
         test_evaluate_with_start_time(input_expr, expected_expr, &Utc::now())
     }
 
-    #[test]
-    fn simplify_expr_not_not() -> Result<()> {
-        let schema = expr_test_schema();
-        let mut rewriter = Simplifier {
-            schemas: vec![&schema],
-        };
+    // ------------------------------
+    // ----- Simplifier tests -------
+    // ------------------------------
 
-        assert_eq!(
-            (col("c2").not().not().not()).rewrite(&mut rewriter)?,
-            col("c2").not(),
-        );
+    // TODO rename to simplify
+    fn do_simplify(expr: Expr) -> Expr {
```

Comment:
👍  and looks ready to rename?

Reply:
I actually have plans to rename it in a follow on PR (b/c I want to remove `simpify`) -- you can see what I have in mind here: https://github.com/apache/arrow-datafusion/pull/1401